drivers/watchdog/sunxi-twd.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#include <error.h>
+#include <mmio.h>
+#include <drivers/clock.h>
+#include <drivers/timer.h>
+#include <drivers/watchdog.h>
+#include <drivers/watchdog/sunxi-twd.h>
+
+#define CLOCK_CYCLES_MHZ    24
```

Comment:
(Just a reminder that with the removal of the interval calculation, this is no longer needed)